src/main/webapp/user-page.html

```diff
@@ -30,14 +30,20 @@
         <li><a href="/">Home</a></li>
       </ul>
     </nav>
-    <h1 id="page-title">User Page</h1>
+    <center>
+      <div id="user-name"><h1 id="page-title">User Page</h1></div>
+    </center>
 
-    <b>About Me:</b>
-    <div id="about-me-container">Loading...</div>
-    <br/>
+    <center>
+      <br/>
+      <b>About Me</b>
```

Comment:
You shouldn't need the br tags. You should be able to use CSS to create the desired margins/padding. Also we should probably be using a h2 instead of a b tag here because this really is a second level header. If you did that then you could target h2 tags with your CSS.